frontend/src/css/NotificationWall.styl

```diff
@@ -37,16 +37,27 @@
         margin standardSpacing
         display inline-block
         width 30px
-      &__text
+       &__text
```

Comment:
Is the tabulation right? It seems that there is one space added.

Reply:
It was a mistake, thanks